packages/react-core/src/components/Panel/PanelMainBody.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
```

Comment:
@mcoker will we ever be able to have more than one `PanelMainBody` in the `PanelMain`?  Or will we ever want to put anything else inside `PanelMain`. if the answer is no to both questions, I don't know that is is necessary to expose this class to consumer.  We can just render the children on `PanelMain` in `PanelMainBody` internally.

Reply:
@tlabaj you can put more than one `PanelMainBody` in `PanelMain`, though it wasn't designed for that. We use that in other places, like cards - https://www.patternfly.org/v4/components/card#with-multiple-body-sections

I'm not sure we will ever want to put anything else in `PanelMain`, but it's worth noting you should be able to use `PanelMain` on its own (without `PanelMainBody`) to get an empty panel without padding (the card also supports that)